### PR TITLE
Update GoCD module and Job

### PR DESCRIPTION
**Pull request by hazim1093**

### CODE REVIEW

File: modules/base_instance/base_instance.tf [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/* VPN Server -> Creating an EC2 instance on AWS */
```
Comment (rasheedamir) [inline modules/base_instance/base_instance.tf]: @hazim1093 seems like copy pasted comment!

**Comment by rasheedamir** [comment]
@hazim1093 shouldn't these kind of files be git ignored?
base_instance.tf.template_backup

File: modules/gocd/conf/cruise-config.xml [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+        <param name="BUILD_DOCKER_IMAGE_NAME"></param>
+        <param name="BUILD_DOCKER_IMAGE_OPTS"></param>
+        <param name="BUILT_WAR_FILE"></param>
+        <param name="APP_COMPILE_CMD"></param>
+        <param name="APP_TEST_CMD"></param>
+        <param name="APP_PACKAGE_CMD"></param>
+        <param name="APP_TEST_ARTIFACT">/</param>
+        <param name="APP_IMAGE_BUILD_DIR"></param>
+        <param name="APP_IMAGE_BUILD_VERSION">${GO_PIPELINE_COUNTER}.${GO_STAGE_COUNTER}</param>
+        <param name="APP_NAME">MyApplication</param>
```
Comment (rasheedamir) [inline modules/gocd/conf/cruise-config.xml:18]: @hazim1093 hard coded name?

File: modules/gocd/conf/cruise-config.xml [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        <param name="APP_DOCKER_IMAGE"></param>
+        <param name="APP_DOCKER_OPTS"></param>
+        <param name="STAKATER_LOCATION">/app/stakater_${GO_PIPELINE_NAME}</param>
+        <param name="STAKATER_COMMAND">make ami_baker;make destroy_base_instance;</param>
+        <param name="STAKATER_CLUSTER_NAME">coreos-cluster</param>
```
Comment (rasheedamir) [inline modules/gocd/conf/cruise-config.xml:23]: @hazim1093 hard coded cluster name? or its a variable ...

File: modules/gocd/conf/cruise-config.xml (hunk 2 above)
Comment (hazim1093) [inline modules/gocd/conf/cruise-config.xml:18]: This is a sample name, the users are supposed to fill up all these params according to their needs

File: modules/gocd/conf/cruise-config.xml (hunk 3 above)
Comment (hazim1093) [inline modules/gocd/conf/cruise-config.xml:23]: it's a variable, the value is a sample value

**Pull request merged**